solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1523 @@
+        // slither-disable-next-line reentrancy-no-eth
+        bridge.updateMovingFundsParameters(
+            movingFundsTxMaxTotalFee,
+            movingFundsDustThreshold,
+            movingFundsTimeoutResetDelay,
+            movingFundsData.getNewMovingFundsTimeout(),
+            movingFundsTimeoutSlashingAmount,
+            movingFundsTimeoutNotifierRewardMultiplier,
+            movedFundsSweepTxMaxTotalFee,
+            movedFundsSweepTimeout,
+            movedFundsSweepTimeoutSlashingAmount,
+            movedFundsSweepTimeoutNotifierRewardMultiplier
+        );
+        movingFundsData.finalizeMovingFundsTimeoutUpdate(governanceDelays[0]);
+    }
+
+    /// @notice Begins the moving funds timeout slashing amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newMovingFundsTimeoutSlashingAmount New moving funds timeout slashing amount.
+    function beginMovingFundsTimeoutSlashingAmountUpdate(
+        uint96 _newMovingFundsTimeoutSlashingAmount
+    ) external onlyOwner {
+        movingFundsData.beginMovingFundsTimeoutSlashingAmountUpdate(
+            _newMovingFundsTimeoutSlashingAmount
+        );
+    }
+
+    /// @notice Finalizes the moving funds timeout slashing amount update process.
+    /// @dev Can be called only by the contract owner, after the governance
+    ///      delay elapses.
+    function finalizeMovingFundsTimeoutSlashingAmountUpdate()
+        external
+        onlyOwner
+    {
+        (
+            uint64 movingFundsTxMaxTotalFee,
+            uint64 movingFundsDustThreshold,
+            uint32 movingFundsTimeoutResetDelay,
+            uint32 movingFundsTimeout,
+            ,
+            uint256 movingFundsTimeoutNotifierRewardMultiplier,
+            uint64 movedFundsSweepTxMaxTotalFee,
+            uint32 movedFundsSweepTimeout,
+            uint96 movedFundsSweepTimeoutSlashingAmount,
+            uint256 movedFundsSweepTimeoutNotifierRewardMultiplier
+        ) = bridge.movingFundsParameters();
+        // slither-disable-next-line reentrancy-no-eth
+        bridge.updateMovingFundsParameters(
+            movingFundsTxMaxTotalFee,
+            movingFundsDustThreshold,
+            movingFundsTimeoutResetDelay,
+            movingFundsTimeout,
+            movingFundsData.getNewMovingFundsTimeoutSlashingAmount(),
+            movingFundsTimeoutNotifierRewardMultiplier,
+            movedFundsSweepTxMaxTotalFee,
+            movedFundsSweepTimeout,
+            movedFundsSweepTimeoutSlashingAmount,
+            movedFundsSweepTimeoutNotifierRewardMultiplier
+        );
+        movingFundsData.finalizeMovingFundsTimeoutSlashingAmountUpdate(
+            governanceDelays[0]
+        );
+    }
+
+    /// @notice Begins the moving funds timeout notifier reward multiplier update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newMovingFundsTimeoutNotifierRewardMultiplier New moving funds timeout notifier reward multiplier.
```

Comment:
Can we break the description into more lines, for example:
```
    /// @notice Begins the moving funds timeout notifier reward multiplier
    ///         update process.
    /// @dev Can be called only by the contract owner.
    /// @param _newMovingFundsTimeoutNotifierRewardMultiplier New moving funds
    ///         timeout notifier reward multiplier.
    function beginMovingFundsTimeoutNotifierRewardMultiplierUpdate(
    ...
 ```

Reply:
[caa12fa](https://github.com/keep-network/tbtc-v2/pull/329/commits/caa12fa98518fff3ea0aad3a420e624f65647e00)